app/views/teams/show.html.slim

```diff
@@ -1,42 +1,56 @@
-.panel.panel-default
-  .panel-heading
-
-    h5
-      strong
-        '#{@team.name}
-      ' team description
-      small
-        a[data-placement="right"
-          data-toggle="popover"
-          data-container=".panel-heading"
-          data-content="<p>Information about the team.</p>"
-          data-original-title="What's this?"
-          tabindex="0" data-html="true"]
-            i.fa.fa-info-circle
-      - if can_manage_team?(@team)
-      .pull-right
-        button.btn.btn-link.btn-xs.btn-edit-role[
-          value="#{@team.id}",
-          class="button_team_description"]
-          i.fa.fa-pencil
-          | Edit team
-  .panel-body
-    .description
-      - if @team.description.blank?
-        p
-          'No description has been posted yet.
-      - else
-        = markdown(@team.description)
-    .collapse id="change_description_team_#{@team.id}"
-      = form_for @team, remote: true, html: {role: 'form'} do |f|
-        = f.text_area(:description, class: 'form-control', placeholder: html_escape(@team.description))
-        br
+.panel-group
+  .collapse id="update_team_#{@team.id}"
+    = form_for @team, remote: true, html: {role: 'form'} do |f|
+      .panel.panel-default
+        .panel-heading
+          .input-group
+            = f.text_field(:name, class: 'form-control', placeholer: html_escape(@team.name), required: true)
+            .input-group-btn
+              button.btn.btn-link.btn-xs.btn-edit-role[
+                value="#{@team.id}"
+                type="button"
+                class="button_edit_team"]
+                i.fa.fa-close
+                | Edit team
+        .panel-body
+          = f.text_area(:description, class: 'form-control', placeholder: html_escape(@team.description))
```

Comment:
Add `tabindex` to 2. It's a bit of a nitpicking from my side (sorry :P), but this is one of those small additions that will make it friendlier for guys that make a heavy usage of the keyboard like me :)